### PR TITLE
Remove MDL2 from code

### PR DESCRIPTION
**Pull request by Jay-o-Way**
## Description

Let's not use MDL2 :)

## Motivation and Context

Closes: #1252

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/ControlPages/IconElementPage.xaml [hunk 1]
```diff
@@ -32,7 +32,7 @@
                 <StackPanel>
                     <TextBlock Text="Use FontIcon as the icon for a control if you want to specify a Glyph value from a FontFamily. Windows 10 uses the Segoe MDL2 Assets FontFamily and that is what this example is showing." Style="{ThemeResource BodyTextBlockStyle}" Margin="0,0,0,12"  />
```
Comment (chingucoding) [inline WinUIGallery/ControlPages/IconElementPage.xaml:33]: I think we need to update this info here. Maybe just remove it? @niels9001 @karenbtlai thoughts?

Comment (niels9001) [reply WinUIGallery/ControlPages/IconElementPage.xaml:33]: Maybe update this with the explaination about Segoe MDL2 Assets being the default on W10 and Segoe Fluent Icons on W11?

Comment (chingucoding) [reply WinUIGallery/ControlPages/IconElementPage.xaml:33]: Sounds good to me. @jay-o-way do you want to include this in your PR?

Comment (Jay-o-Way) [reply WinUIGallery/ControlPages/IconElementPage.xaml:33]: Hm, actually, the header **explicitly** says "...a glyph from a specific font family..." so in that context I should be reverting this specific example.

**Comment by tashatitova** [comment]
Not sure if we can just remove it like that. We use SymbolThemeFontFamily that redirects to appropriate font for controls that use icons

**Review by tashatitova — CHANGES_REQUESTED** [review]
We can't just remove it like that. We use SymbolThemeFontFamily that redirects to appropriate font for controls that use icons. Otherwise it will fall back to Xaml Auto Font Family which is a text font. Also, all these instances would need to be tested, for example, AnimatedIcon

**Comment by Jay-o-Way** [comment]
@tashatitova that's highly peculiar because this very app shows examples just like this. \<Fonticon glyph="">

**Comment by Jay-o-Way** [comment]
Can you show how it looks on your device?

**Comment by Jay-o-Way** [comment]
Actually, all that hard-coding is bad practice. The variable(s) exist(s) for a reason and, as you say, Fonticon defaults to that, i see no reason to hardcode it. Only adding ore possilities for failures.

**Comment by Jay-o-Way** [comment]
@tashatitova 

> FontIcon defaults to SymbolThemeFontFamily

This speaks for itself.

> isn't something recommended, at least as far as I've seen

Have a look at WinUI 3 Gallery, it's a good source:

```xaml
<DropDownButton AutomationProperties.Name="Email">
    <DropDownButton.Content>
        <FontIcon Glyph="&#xE715;"/>
    </DropDownButton.Content>
    <DropDownButton.Flyout>
        <MenuFlyout Placement="Bottom">
            <MenuFlyoutItem Text="Send">
                <MenuFlyoutItem.Icon>
                    <FontIcon Glyph="&#xE725;"/>
                </MenuFlyoutItem.Icon>
            </MenuFlyoutItem>
            <MenuFlyoutItem Text="Reply">
                <MenuFlyoutItem.Icon>
                    <FontIcon Glyph="&#xE8CA;"/>
                </MenuFlyoutItem.Icon>
```

**Comment by Jay-o-Way** [comment]
@niels9001 I'm ready

**Review by niels9001 — APPROVED** [review]
Thanks, changes look good.

**Pull request merged**